components/NavBar.vue

```diff
@@ -81,8 +81,8 @@ export default {
   },
   data() {
     return {
-      visible: screen.width > 768 ? 'hidden' : 'visible',
-      opacity: screen.width > 768 ? '0' : '1',
+      visible: screen.width > 1024 ? 'hidden' : 'visible',
```

Comment:
Request from JP:  can the menu be shown automatically? so that they see the mlh banner and the mentor button, at the moment they have to scroll down and then up to see it!

I think we can just set visible to visible and opacity to 1 by default in that case

Reply:
Thanks for the suggestions, it works! I also removed some code that didn't seem to affect the behavior.